app/src/pages/inside/dashboardPage/dashboardList/dashboardTable/dashboardTable.jsx

```diff
@@ -0,0 +1,231 @@
+  const { onDelete, currentUser: { userId, userRole } } = customProps;
+  const { owner } = value;
+  const deleteItemHandler = () => {
+    onDelete(value);
+  };
+
+  return (
+    <div className={cx('cell', 'with-button', 'delete')}>
+      <div className={cx('icon-holder')}>
+        {(userId === owner || hasPrevilegesForDashboardDeletion(userRole)) && (
+          <Icon type="icon-delete" onClick={deleteItemHandler} />
+        )}
+      </div>
+    </div>
+  );
+};
+DeleteColumn.propTypes = {
+  value: PropTypes.object,
+  customProps: PropTypes.object,
+};
+DeleteColumn.defaultProps = {
+  value: {},
+  customProps: {},
+};
+
+@injectIntl
+export class DashboardTable extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    onDeleteItem: PropTypes.func,
+    onEditItem: PropTypes.func,
+    onAddItem: PropTypes.func,
+    userInfo: PropTypes.object,
+    dashboardItems: PropTypes.array,
+  };
+
+  static defaultProps = {
+    onDeleteItem: () => {},
+    onEditItem: () => {},
+    onAddItem: () => {},
+    userInfo: {},
+    dashboardItems: [],
+  };
+
+  getTableColumns() {
+    const { onDeleteItem, onEditItem, userInfo, intl } = this.props;
+
+    return [
+      {
+        title: {
+          full: intl.formatMessage(messages.dashboardName),
+          short: intl.formatMessage(messages.dashboardName),
+        },
+        component: DashboardNameColumn,
+      },
+      {
+        title: {
+          full: intl.formatMessage(messages.description),
+          short: intl.formatMessage(messages.description),
+        },
+        component: DescriptionColumn,
+        formatter: (value) => value.description,
+      },
+      {
+        title: {
+          full: intl.formatMessage(messages.owner),
+          short: intl.formatMessage(messages.owner),
+        },
+        formatter: (value) => value.owner,
+        component: OwnerColumn,
+      },
+      {
+        title: {
+          full: intl.formatMessage(messages.shared),
+          short: intl.formatMessage(messages.shared),
+        },
+        component: SharedColumn,
+        customProps: {
+          currentUser: userInfo,
+        },
+      },
+      {
+        title: {
+          full: intl.formatMessage(messages.edit),
+          short: intl.formatMessage(messages.edit),
+        },
+        component: EditColumn,
+        customProps: {
+          onEdit: onEditItem,
+          currentUser: userInfo,
+        },
+      },
+      {
+        title: {
+          full: intl.formatMessage(messages.deleteDashboard),
+          short: intl.formatMessage(messages.deleteDashboard),
+        },
+        component: DeleteColumn,
+        customProps: {
+          onDelete: onDeleteItem,
+          currentUser: userInfo,
+        },
+      },
+    ];
+  }
+
+  COLUMNS = this.getTableColumns();
+
+  render() {
+    const { dashboardItems, onAddItem } = this.props;
+
+    return (
+      <Fragment>
+        <Grid
+          className={cx('dashboard-table')}
```

Comment:
Can this be achieved by wrapping Grid with a div with a class name?

Reply:
Unfortunately no, because grid component by default has boxshadow, so it looks not as per designs